src/components/autocomplete/js/autocompleteController.js

```diff
@@ -601,6 +601,8 @@ function MdAutocompleteCtrl ($scope, $element, $mdUtil, $mdConstant, $mdTheming,
     if (!noBlur) {
       ctrl.hidden = shouldHide();
       evalAttr('ngBlur', { $event: $event });
+    } else {
+      $event.stopImmediatePropagation();
```

Comment:
```suggestion
    } else if (angular.isObject($event)) {
      $event.stopImmediatePropagation();
    }
```
```

Reply:
Thank you! I made a commit with your suggestion, but I don't quite understand this test `shows the md-not-found template even if we have lost focus` in autocomplete.spec.js, why after `controller.blur();` we expecting that notFoundTemplate should be visible?
```
controller.blur();
expect(controller.notFoundVisible()).toBe(true);
```

Reply:
And one more question, should I add some tests for my changes?

Reply:
You can see the reasoning behind the test using Git Blame which points to this commit:
https://github.com/angular/material/commit/52a519ead04f2faeba227fbc51a7214253cbf2ba

It explains that the main use case here is actually clicking on the not found item in the dropdown. It shouldn't dismiss the panel. The test could certainly have a bit better description.

Reply:
I don't think that we need a test for this. I was just able to verify this manually in the demo on the docs site.

Reply:
Ok, thanks for the explanation